tests/integration-tests/Cargo.toml

```diff
@@ -80,6 +80,7 @@ dns-over-openssl = [
 ]
 dns-over-rustls = [
     "dns-over-tls",
+    "dnssec-ring",
```

Comment:
How do chained authorities relate to DNSSEC? Is DNSSEC really a necessary prerequisite?

Reply:
Intrinsically? The two concepts are completely unrelated.  As the server crate dependencies are currently written:

```
dns-over-rustls = [
     "dns-over-tls",
     "dnssec-ring",
...
```

  Leaving this out causes the dns-over-rustls test suite to fail due to missing trait items:

```
error[E0046]: not all trait items implemented, missing: `get_nsec3_records`, `nx_proof_kind`
   --> tests/integration-tests/tests/integration/chained_authority_tests.rs:198:1
    |
198 | impl Authority for TestAuthority {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `get_nsec3_records`, `nx_proof_kind` in implementation
    |
    = help: implement the missing item: `fn get_nsec3_records(&'life0 self, _: Nsec3QueryInfo<'life1>, _: LookupOptions) -> Pin<Box<(dyn futures::Future<Output = LookupControlFlow<<Self as Authority>::Lookup>> + std::marker::Send + 'async_trait)>> { todo!() }`
    = help: implement the missing item: `fn nx_proof_kind(&self) -> Option<&NxProofKind> { todo!() }`
```

Reply:
Okay, fair enough. We should probably try to fix that in a cleaner way at some point, but this is fine for now.